### ISSUE TITLE
Default evolution tempo hasn't changed since 6 to 1s change. 

### ISSUE THREAD

**Issue by SirPendrak** [CleverRaven/Cataclysm-DDA#35963]
Frist, I know i can change it during worldgen to match my needs. But that doesnt mean default value shouldnt give the best possible and ballanced gameplay. 

CDDA almost always suffered from issue of too harsh starts and too easy mid-lategame. There are some recent changes in good direction (EVAC rework, removing advanced Zeds from early spawns, adding bone and kevlar hulks) but there is still a lot to do. Ideally, a good game should have an easy start and difficult part after it. I mean difficult to a degree that if youre not learning the game, it wrecks you. 
So if a new player starts playing CDDA he gets a few days to figure everything out and secure starting area. But after that every raid should be dangerous and possibly end many characters life. Then you start new game and can use your knowledge to be better prepared. 
In past, most people would die too fast and didnt have chance to learn anything from their deaths, and those who managed to survive the beginning, quickly ended in post-apo paradise for their mutated cyborgs demigod fighting masters where they can go everywhere and take everything while smashing some skulls on the way. 

But back to evolution tempo. Turns taking 1/6th of what used to take before do not mean you can do 6x as much stuff in the same time, but you can do 2-3 as much things depending on your playstyle (more fighting, traveling and scavenging or more reading, building and crafting). And basic evolution tempo established after evolution rework was already slow. 
In vanilla game, you can get your 8/8/8/8 character with 0 starting skills to 3-5 in fighting skills, 7 or sometimes 11 dodge and have light survivor gear in 20 days. Some basic melee weapon like machette and some ranged weapon as well. Ive done this many times, and at this level you are already ready to melee single hulk. 
Your character can advance about 10x times as fast in power lvl as average zombie if you focus only on lvling skills and getting better gear. And you can advance further with bionics and mutations than zeds. Killing one kevlar hulk is easier for 11 all skill demigod than killing one standard zed for fresh newbie. Same goes for hordes, lategame charcacter can defeat evolved horde more easily than new player can defeat mundane horde. And you get more options in lategame like luring things out to safe area, sniping things, building indefeatable deathmobiles, getting nearly infinite ammo with production or solar chargers for lasers etc.
My suggestion would be to change default evolution tempo form 4.0 to 1.0. For reasons written above. And for players who dont do rlly long playthroughs to see some more interesting enemies that they otherwise would never encountered.

**Comment by Amoebka** [comment]
Crafting and reading - the two main ways characters progress in power still take a lot of time and it's planned for them to take even longer. Moving is 6 times faster, but reading a book is exactly the same as before.
The problem with midgame isn't the evolution, it's the lack of incentives to take risks. Early in the game you HAVE to raid and loot to survive. Later, you have everything you need to last for centuries and raid because you CAN. When you aren't forced to go to dangerous places you can make sure you only go where you know will be safe for your current level.
I'm playing with 1.0 evolution, and I can tell it's not the solution. Sure, cities will be deadly 20 days in, but why would you possibly bother with cities at that point? After 20 days I tend to have post-threshold mutants with filled bionic slots even of non-fast starts, am I supposed to raid clothing stores? Evolution makes locations with lots of weak enemies deadly, but those are usually the places that don't have loot. Late game dungeons (which is currently labs and nothing else) don't and shouldn't have a lot of weak enemies, so they aren't affected by evolution all that much.

TLDR: Evolution won't solve late game issues, more mapgen content will.

**Comment by SirPendrak** [comment]
I am not saying that raising evolution tempo will solve all problems, i am saying that it should be changed because you can do now 2-3 times as much as before in same ammount of time and it was a bit too low before.

**Comment by Amoebka** [comment]
| because you can do now 2-3 times as much as before
But you can't. The absolute majority of time isn't spend walking, it's spent reading, crafting or sleeping - all of which take the exact same amount of time as before the turn change.
Especially early in the game. On your first day you run around for maybe 3 hours, then the chip damage from zombies adds up to unsafe levels and your focus drops to zero, so you spend the rest of the day reading, sleeping and waiting for your wounds to heal.

**Comment by Zupercharged** [comment]
The main issue with just raising tempo is that it doesn't address your problem. As you laid it out the big issue was that on top of late game being easy _early game was hard_, especially on new players. Upping the tempo is just more likely to see them die before learning anything making it harder for them to get into the game, hence why its better for more experienced/confident players to just up It themselves in the options over chaning the base level.

To add to what Amoebka was saying earlier, the big reason why stuff feels easy once you pass the point where youre prepared is that you can take things at your own pace and pick your targets. More varied content across the world map is one way to keep things interesting, but the other is more dynamic and engaging content. More indepth NPC interactions, more mobile encounters, heck even more interesting terrain would spice things up. Upping the default difficulty with a mind to late game content is a bit long sighted, considering the people most effected by this are the people who don't know what these settings mean.

**Comment by kevingranade** [comment]
Addressed by #36821 
If that is insufficient need a stronger and more detailed rationale for why it needs to be increased.
